firebase-app/src/androidMain/kotlin/dev/gitlive/firebase/firebase.kt

```diff
@@ -53,7 +53,7 @@ actual class FirebaseOptions actual constructor(
         actual fun withContext(context: Any): FirebaseOptions? {
             return when (context) {
                 is Context -> com.google.firebase.FirebaseOptions.fromResource(context)
-                else -> com.google.firebase.FirebaseOptions.Builder().build()
```

Comment:
`FirebaseOptions.Builder` shall be used with application id, it always fails otherwise

Reply:
Is it stays in docs?
Also `context` is `Any` so it can be an application id, so we can add this case as well?

Reply:
Ah I see, it fill fallback into `run`, never mind then